src/main/java/racingcar/model/Car.java

```diff
@@ -1,28 +1,31 @@
 package racingcar.model;
 
-import racingcar.util.RandomNumberGenerator;
-
-public class Car {
-	private String name;
+public final class Car {
+	private final String name;
 
 	public String name() {
 		return name;
 	}
 
-	private Integer distance;
+	private Distance distance;
 
-	public Integer distance() {
+	public Distance distance() {
 		return distance;
 	}
 
-	public Car(String name) {
+	public Car(final String name) {
+		this.name = name;
+		this.distance = new Distance(0);
```

Comment:
```suggestion
		this(name, new Distance(0));
```
부생성자가 주생성자를 호출하도록 구현할 것을 추천